src/dev/flang/fe/SourceModule.java

```diff
@@ -945,7 +946,9 @@ else if (c._hasPost != null && c._hasPostThen == null)
 A feature that does not redefine an inherited feature must not use the `redef` modifier.
     // end::fuzion_rule_PARS_NO_REDEF[]
             */
-            AstErrors.redefineModifierDoesNotRedefine(f);
+            List<FeatureAndOuter> hiddenFeaturesSameSignature = lookup(outer, f.featureName().baseName(), null, true, true);
+            hiddenFeaturesSameSignature = hiddenFeaturesSameSignature.stream().filter(fan->fan._feature.featureName().equals(f.featureName())).collect(List.collector());
```

Comment:
Is there a reason you did not use `FeatureAndOuter.filter`?

Reply:
I found it easier. I did not fully understand all the arguments `FeatureAndOuter.filter` requires and also it produces an error if there is more than one match.

Reply:
@simonvonhackewitz okay, makes sense.